modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -49,42 +49,44 @@
  */
 function editIssue()
 {
-    $db =& Database::singleton();
+    $db   =& Database::singleton();
     $user =& User::singleton();
 
-    $issueValues = array();
+    $issueValues    = array();
     $validateValues = array();
-    $fields = array(
-        'assignee',
-        'status',
-        'priority',
-        'centerID',
-        'title',
-        'category',
-        'module',
-    );
+    $fields         = array(
+                       'assignee',
+                       'status',
+                       'priority',
+                       'centerID',
+                       'title',
+                       'category',
+                       'module',
+                      );
     $fieldsToValidateFirst = array(
-        'PSCID',
-        'visitLabel',
-    );
+                              'PSCID',
+                              'visitLabel',
+                             );
 
     foreach ($fields as $field) {
-        if (isset($_POST[$field])) {
-            $issueValues[$field] = $_POST[$field];
+        $value = $_POST[$field];
+        if (strcmp($_POST[$field], "null") == 0) {
```

Comment:
No particular reason, aside from maybe making it explicitly clear that we check for a "null" string